wiki/markdown/running-eiffel-intelligence.md

```diff
@@ -26,12 +29,18 @@ With properties added to the maven command, e.g:
     
     mvn spring-boot:run -Dlogging.level.com.ericsson.ei=DEBUG -Dspring.data.mongodb.port=27019
 
-Or you can run the executable war file (located in target folder, if running from source code):
+ ## Running with java command 
 
-    java -jar eiffel-intelligence-<version>.war
+Another option is to run the executable war file with java command.
+If running from source code, war file is generated and produced by maven command (mvn install command can be used as well):
+
+    mvn package -DskipTests
 
-if you want to run with default configuration for local message buss and
-Mongo DB.
+ This command should produce a eiffel-intelligence-<version>.war file in target folder, target/eiffel-intelligence-<version>.war. 
+
+War file is executed by following command and with default configuration: 
```

Comment:
The war file is executed by the following command.
Occasionally 'a' and 'the' are omitted.

Reply:
Fixed.